From 5e135435a1f69face856e27110b5be0f477a06c1 Mon Sep 17 00:00:00 2001
From: Sven Neumann <s.neumann@raumfeld.com>
Date: Tue, 19 Apr 2011 12:07:52 +0200
Subject: [PATCH 1/2] Add quotes around charset value in HTTP header

The UPnP spec requires quotes around the charset value in the
Content-Type HTTP header for action responses.
---
 libgupnp/gupnp-service-proxy.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/libgupnp/gupnp-service-proxy.c b/libgupnp/gupnp-service-proxy.c
index 3400576..c976c9b 100644
--- a/libgupnp/gupnp-service-proxy.c
+++ b/libgupnp/gupnp-service-proxy.c
@@ -787,7 +787,7 @@ finish_action_msg (GUPnPServiceProxyAction *action,
                          "</s:Envelope>");
 
         soup_message_set_request (action->msg,
-                                  "text/xml; charset=utf-8",
+                                  "text/xml; charset=\"utf-8\"",
                                   SOUP_MEMORY_TAKE,
                                   action->msg_str->str,
                                   action->msg_str->len);
-- 
1.7.1

